clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5303,6 +5305,35 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         D.addSYCLTargetMacroArg(Args, Macro);
       }
     };
+    for (const auto &entry : DeviceConfigFile::TargetTable) {
```

Comment:
This piece of code here is just to illustrate how to use the Device Config file. I'll remove this when this moves out of draft state.